src/azure-cli/azure/cli/command_modules/acr/taskrun.py

```diff
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._utils import (
+    validate_managed_registry
+)
+
+
+TASKRUN_NOT_SUPPORTED = 'TaskRun is only supported for managed registries.'
+
+
+def acr_taskrun_show(cmd,
+                     client,
+                     taskrun_name,
+                     registry_name,
+                     resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
```

Comment:
can we use validator for you command or argument https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#validators?

Reply:
@Juliehzl  current `validate_managed_registry` also returns the resource group to be used in the subsequent codes. It's also used in other acr commands. So I think it's out of the scope of current pull request. It's better we evaluate the validator option later for acr commands together. /cc: @djyou @yugangw-msft